packages/blocks/src/page-block/doc/doc-page-block.ts

```diff
@@ -224,6 +227,26 @@ export class DocPageBlockComponent extends BlockElement<
         })
       );
     });
+
+    const scrollElement = this.closest(
+      'div.affine-doc-viewport'
```

Comment:
Can we use the nearest parent scroll container here to avoid relying on external implementations?

Reply:
em, after thinking, I feel this feature better be implemented in affine side.  Because now blocksutie provides components, then composed in affine.